ovirt_imageio/client/_options.py

```diff
@@ -10,62 +10,124 @@
 Tool options.
 """
 
+from collections import namedtuple
+
 from .. _internal.units import KiB, MiB, GiB, TiB
 
 import argparse
 import configparser
 import os
+import sys
+
+
+class Option(
+        namedtuple("Option", "name,args,config,required,type,default,help")):
```

Comment:
I think it would be good to have a comment here, to describe what the fields mean, esp. "args" and "config".

Reply:
The arguments are documented now.